CMakePresets.json

```diff
@@ -151,7 +151,9 @@
         "quiche_ROOT": "/opt/h3-tools-boringssl/quiche",
         "CMAKE_INSTALL_PREFIX": "/tmp/ats-quiche",
         "opentelemetry_ROOT": "/opt",
-        "ENABLE_QUICHE": "ON"
+        "ENABLE_QUICHE": "ON",
+        "CMAKE_CXX_FLAGS_DEBUG": "-g -fsanitize=address",
+        "CMAKE_C_FLAGS_DEBUG": "-g -fsanitize=address"
```

Comment:
We should be able to use `"ENABLE_ASAN": "ON",` instead of setting the flags. That would be preferable. Can you also use `ENABLE_ASAN` for the other places in here that set those `-fsanitize=address` flags?

Reply:
Sure. For the other places, I'll open another PR.